.github/workflows/ci.yml

```diff
@@ -25,26 +25,29 @@ jobs:
         with:
           node-version: ${{ env.NODE_VERSION }}
 
-      - name: Get Yarn cache path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: Update npm
+        run: npm i -g npm@latest
 
-      - name: Cache Yarn cache and node_modules
+      - name: Get npm cache path
+        id: npm-cache-dir-path
+        run: echo "::set-output name=dir::$(npm config get cache)"
+
+      - name: Cache npm cache and node_modules
         id: cache-dependencies
         uses: actions/cache@v2
         with:
           path: |
-            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+            ${{ steps.npm-cache-dir-path.outputs.dir }}
```

Comment:
[Before the change](https://github.com/ember-learn/ember-website/pull/698/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fL33), we just had `~/.npm` (a constant). Can you check if we can do without and remove lines 31-35?

Reply:
so I just double checked the documentation that I took this from and it recommends that we use the structure that we have now https://github.com/actions/cache/blob/main/examples.md#node---npm 

I think it's probably best to go with the recommended config 👍